api/src/test/resources/io/strimzi/api/kafka/model/KafkaConnector.out.yaml

```diff
@@ -10,4 +10,19 @@ spec:
   tasksMax: 2
   config:
     file: "/home/source/test.source.txt"
-    topic: "test.topic"
\ No newline at end of file
+    topic: "test.topic"
+status:
+  observedGeneration: 0
+  connector:
+    name: "hdfs-sink-connector"
+    connector:
+      state: "RUNNING"
+      worker_id: "whatever:8083"
```

Comment:
this is quite related to what I mentioned in the comment on the PR. Why having `connector.connector.state` (or `.worker_id`) instead of just `connector.state` (or `.worker_id`)?

Reply:
The first `connector` is the name of the property in our schema (which we could rename `connectorStatus` or anything else which we can agree on). The second `connector` is the name of a property in the Connect API response which we have no control over.